stats/Metrics.md

```diff
@@ -0,0 +1,19 @@
+# Metrics
+Metrics are a data model for what stats exporters take as input.
+
+Different exporters have different capabilities (e.g. which data types
+are supported) and different constraints (e.g. which characters are allowed in
+label keys). Metrics is intended to be a superset of what's possible, not a
+lowest common denominator that's supported everywhere.
+
+Because of this, Metrics puts minimal constraints on the data (e.g. which
+characters are allowed in keys), and code dealing with Metrics should avoid
+validation and sanitization of the Metrics data. Instead, pass the data to the
+backend, rely on the backend to perform validation, and pass back any errors
+from the backend.
+
+The Metrics data model is defined as
+[metrics.proto](https://github.com/census-instrumentation/opencensus-proto/blob/master/opencensus/proto/stats/metrics/metrics.proto),
+but the proto is just to illustrate the concepts. OpenCensus implementations
+don't have to use the actual proto, and can instead use a language-specific
+in-memory data structure (like `ViewData`) that captures what exporters need.
```

Comment:
> language-specific
in-memory data structure (like `ViewData`)

I think that the in-memory data structures that are part of the API should still be required to use the names and fields from the Protocol Buffer, for consistency in the API across languages.  For example, I think opencensus-java will have a `Metric` class with a similar API to the class that would have been generated from the Protocol Buffer.

Reply:
I've added wording to this effect. Please take a look.